### PR TITLE
List of tags for the home page

### PR DESCRIPTION
**Pull request by SimonLab**
#28 #57 Hi @NataliaLKB please review this pull request. Thanks

I have added the riot tags for login and for home page, I realise that some of the names of the variables are going to have to change in accordance with the actual issues object that gets returned by the server.

### CODE REVIEW

File: views/tags/login.tag [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<login>
```
Comment (NataliaLKB) [inline views/tags/login.tag:1]: @SimonLab is this the latest version of login.tag? Do double check and get back to me.

**Comment by NataliaLKB** [comment]
@SimonLab would you mind updating the README explaining what in this code is definitely changing and what is good to start working with?

Also, are the empty files just plans for the future? Maybe consider [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

**Comment by SimonLab** [comment]
@NataliaLKB I update the login page to the last version, delete the empty files. I also upate the Readme with a short description for the components. You can now use and render the container_home tag in a new handler.

**Comment by NataliaLKB** [comment]
@SimonLab is this ready for us to start integrating with yet?

**Comment by SimonLab** [comment]
@NataliaLKB yes all the tags are ready to be used.

**Comment by SimonLab** [comment]
@jrans Your test and mine passed together. The branch contains the components for the login page, the home page. The test utilise the fixtures files created by @jrans

**Comment by jrans** [comment]
Thanks @SimonLab  & @benjaminlees . Done dwyl proud.

**Pull request merged**